ProjectSettings/EditorBuildSettings.asset

```diff
@@ -5,6 +5,9 @@ EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
   m_Scenes:
+<<<<<<< HEAD
```

Comment:
looks like there were some mistakes introduced unintentionally. I'll fix these. 

for now, if you need to continue working, make a branch from here.